app/javascript/components/pages/PastMeetup.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { getPastMeetup } from '../../datasources';
+import SharedLayout from 'components/layout/SharedLayout';
+import PageTitleWithContainer from 'components/PageTitle';
+import SpeakersList from '../SpeakersList';
+import Microphone from '../icons/Microphone';
+import LoadingSpinner from 'components/LoadingSpinner';
+import 'stylesheets/page';
+import 'stylesheets/meetup';
+
+const VideoBlock = ({ videoUrl, title }) => {
+    if (!videoUrl) {
+        return null;
+    }
+    return (
+        <div className="card-container flex flex-wrap justify-center aspect-w-16 aspect-h-9">
+            <iframe src={videoUrl} title={title} frameBorder="0"></iframe>;
+        </div>
+    );
+};
+
+VideoBlock.propTypes = {
+    videoUrl: PropTypes.string,
+    title: PropTypes.string,
+};
+
+const SpeakerBiosBlock = ({ speakers }) => {
+    return (
+        <div className="container max-w-2xl my-8 mx-3 p-4 flex flex-col">
+            <div className="inline-flex items-center gap-2 align-center mb-5">
+                <Microphone />
+                <h4 className="text-xl font-bold text-gray md:text-2xl">About the speakers</h4>
+            </div>
+            <div className="flex flex-wrap items-center gap-5">
+                {speakers?.map(({ id, bio }) => (
+                    <div key={id}>{bio}</div>
+                ))}
+            </div>
+        </div>
+    );
+};
+
+SpeakerBiosBlock.propTypes = {
+    speakers: PropTypes.arrayOf(PropTypes.object),
+};
+
+const PastMeetup = () => {
+    const year = window.year;
+    const month = window.month;
+    const eventDate = new Date(Date.UTC(year, Number(month) - 1));
```

Comment:
```suggestion
    const eventDate = new Date(Date.UTC(year, Number(month)));
```

I'm still experiencing the off-by-one month error locally, and I no longer experience it if I remove this `- 1`. Can you tell me more about why we need the `- 1` here?

Reply:
Noting here for posterity - this was working for me in the UK, but not for @egiurleo, who cleverly worked out that the `Date.UTC` was causing a timezone bug!  I have now removed that code and tested here in the UK and it works for me.